tests/test_temperature.py

```diff
@@ -1398,8 +1398,8 @@ def test_corn_heat_units(open_dataset):
         tnC = tn - K2C
         tnC.attrs["units"] = "C"
 
-    chu = atmos.corn_heat_units(tasmin=tn, tasmax=tx, thresh_tasmin="4.44 degC", thresh_tasmax="10 degC")
-    chuC = atmos.corn_heat_units(tasmin=tnC, tasmax=tx, thresh_tasmin="4.44 degC", thresh_tasmax="10 degC")
+    chu = convert.corn_heat_units(tasmin=tn, tasmax=tx, thresh_tasmin="4.44 degC", thresh_tasmax="10 degC")
+    chuC = convert.corn_heat_units(tasmin=tnC, tasmax=tx, thresh_tasmin="4.44 degC", thresh_tasmax="10 degC")
```

Comment:
hum, I think `corn_heat_units` should not be in convert. It has thresholds and all that. It should go back in agro  IMO

Reply:
I agree. I need to see what can be done about the Indicator version of this indice though; It's currently a `Converter` there, using CF-checking. Perhaps that can be changed without breaking anything.